### ISSUE TITLE
Dementia Report Table

### ISSUE THREAD

**Issue by grosscol** [Landis-Lewis-Lab/goals-of-care#63]
Add a summary table **soley to the dementia report template**.

Table with six month columns oldest to most recent from left to right.

Table with four groups of rows:
- Short Stay – All patients
- Short Stay –Patients with DEMENTIA
- Long-term care – ALL residents
- Long-term care – Residents with DEMENTIA

The Short Stay groups each have the following data rows:
- Total number pts
- Number (%) with new LST this month
- Number (%) with existing LST this month
- Number (%) with any LST

The Long-term care groups each have the following data rows:
- Total number residents
- Number (%) with new LST this month
- Number (%) with existing LST this month
- Number (%) with any LST

`Number (%)` values should display the indicated numerator followed by a percentage consisting of the in
dicated numerator divided by the `Total number` for the row group.  E.g. 23 (75%)

**Comment by grosscol** [comment]
There are three categories of numerator for dementia reports:
```R
list("cat_1"=c("goc_none"),
       "cat_2"=c("goc_pre_month"),
       "cat_3"=c("goc_first_month"))
```

Calculation of those for each time point to produce the summary table data is as follows:

| label | summation |
|-------|--------------|
|`Total number pts` | `cat_1 + cat_2 + cat_3` |
|`Number (%) with new LST this month` | `cat_3` |
|`Number (%) with existing LST this month` | `cat_2` |
|`Number (%) with any LST` | `cat_2 + cat_3`

**Comment by grosscol** [comment]
Commit c1fdea2d76fe854aa08da3faa779a4e2163fc402 generates the following table:
![dementia_table](https://user-images.githubusercontent.com/1520508/48720626-0a28f400-ebee-11e8-9646-788ac09e0309.png)

**Comment by grosscol** [comment]
Just noticed the title is left over from the previous iteration.  The working title is now "Veterans goals of care documentation"

**Comment by grosscol** [comment]
Subsequent changes to the above draft to make:
> A few, minor suggestions I have (let's see what others think) are to rearrange the rows to read thus:
>Number with a new LST this month
>Number with an existing LST this month
>Number with new/existing LST
>Total number pts
>
>Also, at present the last row uses the term “pts” for long-term care residents. Suggest changing to “residents“

**Comment by grosscol** [comment]
Closed by ee582831025eb4a620dae595562410487f8e67c1
